lib/gems/pending/appliance_console/prompts.rb

```diff
@@ -143,7 +143,7 @@ def ask_for_integer(prompt, range = nil, default = nil)
       just_ask(prompt, default, INT_REGEXP, "an integer", Integer) { |q| q.in = range if range }
     end
 
-    def ask_for_disk(disk_name, verify = true)
+    def ask_for_disk(disk_name, verify = true, force = false)
```

Comment:
I feel like this is using the word "force" in the opposite way I would expect.

If someone calls `#ask_for_disk` I would think that "force" would mean "Give me an answer no matter what". What you're doing here feels more like "Abort if we can't get a disk".

I think this would be better if the parameter had a more clear name, maybe `allow_none`?

Reply:
you're right, thanks